libsolidity/interface/CompilerStack.cpp

```diff
@@ -1217,7 +1217,7 @@ Json CompilerStack::ethdebug(Contract const& _contract, bool _runtime) const
 	evmasm::LinkerObject const& object = _runtime ? _contract.runtimeObject : _contract.object;
```

Comment:
```
if (_contract.contract->isInterface())
	return {};
```

Reply:
You may also need to include abstract contracts and libraries in the check as well.